### PR TITLE
api-extractor: add option to generate API reports for select packages

### PR DESCRIPTION
**Pull request by Rugvip**
## Hey, I just made a Pull Request!

You can now pass package paths to `build:api-reports`, so instead of waiting 2 minutes, it's more like 20 seconds :grin:

image

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "start": "yarn workspace example-app start",
     "start-backend": "yarn workspace example-backend start",
     "build": "lerna run build",
-    "build:api-reports": "yarn tsc:full && yarn build:api-reports:only",
+    "build:api-reports": "yarn build:api-reports:only --tsc",
```
Comment (freben) [inline package.json:12]: hehe, amusingly backwards, but why not :)

**Review by freben — APPROVED** [review]
Alright. Would be awesome if it could automatically do it only for packages that have git changes of course.

**Comment by Rugvip** [comment]
@freben yeah was going for that initially but realized that's very often not really what you want

**Review by jhaals — APPROVED** [review]
Awesome!

**Pull request merged**